### PR TITLE
Report the status of admin items in SSR

### LINKED ISSUE #32583: Report the status of admin items in SSR
To help testers verify the status of back-end items used by the react admin client, it would be nice to have reporting in the [SSR](https://href.li/?https://woocommerce.com/document/understanding-the-woocommerce-system-status-report/).

For example:

* Which features are enabled/disabled?
* Is the daily cron job active, when does it next run?
* Do the important [options](https://href.li/?https://developer.wordpress.org/plugins/settings/options-api/) return expected values?
* How many notes in the database?
* Was onboarding completed or skipped?

Note: the required database tables are already checked in the SSR.

## Example Mockup

This is a quick mockup based on the Action Scheduler report. It doesn't need to look the same.

Screen Shot 2022-04-12 at 4 14 26 pm

### PR DESCRIPTION
**Pull request by chihsuan**
### All Submissions:

-   [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Closes #32583.

Add status of admin items to SSR to help testers verify the status of back-end items.
Screen Shot 2022-05-16 at 15 05 33

### How to test the changes in this Pull Request:

1. Run `pnpm nx composer-install woocommerce`
2. Go to **WooCommerce > Status**
3. Scroll down to "Admin" section
4. Confirm the status items show as expected

### Other information:

-   [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?
-   [ ] Have you successfully run tests with your changes locally?
-   [x] Have you created a changelog file for each project being changed, ie `pnpm nx changelog <project>`?

### FOR PR REVIEWER ONLY:

-   [x] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: plugins/woocommerce/src/Internal/Admin/SystemStatusReport.php [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+		}
+		return self::$instance;
+	}
+
+	/**
+	 * Hook into WooCommerce.
+	 */
+	public function __construct() {
+		add_action( 'woocommerce_system_status_report', array( $this, 'system_status_report' ) );
+
```
Comment (ilyasfoo) [inline plugins/woocommerce/src/Internal/Admin/SystemStatusReport.php]: Nitpick: extra space
```suggestion
```

File: plugins/woocommerce/src/Internal/Admin/SystemStatusReport.php [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+				</td>
+			</tr>
+		<?php
+	}
+
+	/**
+	 * Render option row.
+	 */
+	public function render_options() {
+		$all_options_expected = is_numeric( get_option( 'woocommerce_admin_install_timestamp' ) )
```
Comment (ilyasfoo) [inline plugins/woocommerce/src/Internal/Admin/SystemStatusReport.php]: Small thing but I think it's worth making sure that `woocommerce_admin_install_timestamp` is more than 0

**Review by ilyasfoo — DISMISSED** [review]
This looks great, nice work @chihsuan! Tested well. I think there *may* be more option check that we can add, but I can't find any that's as important. Anyway if I can think of anything, we can always do a follow-up.

Some nitpick comments:

File: plugins/woocommerce/src/Internal/Admin/SystemStatusReport.php (hunk 2 above)
Comment (chihsuan) [reply plugins/woocommerce/src/Internal/Admin/SystemStatusReport.php]: Good point! Updated in 902dcf3369212732fcb9490260660b459e888f08

File: plugins/woocommerce/src/Internal/Admin/SystemStatusReport.php (hunk 1 above)
Comment (chihsuan) [reply plugins/woocommerce/src/Internal/Admin/SystemStatusReport.php]: Removed in 902dcf3369212732fcb9490260660b459e888f08

**Comment by botwoo** [comment]
:bar_chart: Test reports for 902dcf3369212732fcb9490260660b459e888f08 `Check if woocommerce_admin_install_timestamp > 0 for SSR & remove ext…` have been published and are accessible through the following links:

- [REST API Test Report](https://woocommerce.github.io/woocommerce-test-reports/pr/33030/api/)
- E2E Test Report (TBC)

Visit the [WooCommerce Test Reports](https://woocommerce.github.io/woocommerce-test-reports) homepage to view all published reports. See the [FAQs](https://github.com/woocommerce/woocommerce-test-reports/wiki/FAQs) page if you're having problems accessing them.

**Review by ilyasfoo — APPROVED** [review]
Nice, LGTM!

**Pull request merged**